judge/admin/contest.py

```diff
@@ -73,7 +73,7 @@ class ContestProblemInline(SortableInlineAdminMixin, admin.TabularInline):
     def rejudge_column(self, obj):
         if obj.id is None:
             return ''
-        return format_html('<a class="button rejudge-link" href="{}">Rejudge</a>',
+        return format_html('<a class="button rejudge-link" href="{1}">{0}</a>', _('Rejudge'),
```

Comment:
Could we flip the arg order here?

Reply:
Done. The button is still good.

![Capture](https://user-images.githubusercontent.com/14223529/198512119-152f5ed8-a4b5-48ed-a700-e7271fc2d934.PNG)